packages/fast-components-react-base/src/toolbar-item-group/toolbar-item-group.tsx

```diff
@@ -0,0 +1,183 @@
+import { ToolbarItemGroupClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { classNames, Orientation } from "@microsoft/fast-web-utilities";
+import { canUseDOM } from "exenv-es6";
+import React from "react";
+import { DisplayNamePrefix } from "../utilities";
+import {
+    ToolbarItemGroupHandledProps,
+    ToolbarItemGroupProps,
+    ToolbarItemGroupUnhandledProps,
+} from "./toolbar-item-group.props";
+import Toolbar from "../toolbar/toolbar";
+import { isNil } from "lodash-es";
+import { ToolbarContext, ToolbarContextType } from "../toolbar/toolbar-context";
+
+export interface ToolbarItemGroupState {
+    /**
+     *  the indexes of children of this component that can receive focus
+     */
+    focusableItemIndexes: number[] | null;
```

Comment:
Same question as above - are there alternative API's you explored for this functionality?

Reply:
I'm not really worried about this one as it is component internal implementation and not an exposed api.